registry.json

```diff
@@ -4365,5 +4365,18 @@
    "telegram": "HunterKoppen",
    "operatorName": "Hunter Koppen",
    "github": ""
-  }
+  },
+  {
+ "ticker": "RCKSP",                                                       
+ "poolId": "435f7f1af165a0696f8142f8f2863f2e2e71c5cab269d264ad5b6367",  
+ "social": {                                                           
+   "twitter": "Rckstr_io",
+   "facebook": "https://rckstr.io/",  
+   "instagram": " ",
+   "youtube": " "
+ },
+ "telegram": "rckstr.io [RCKSP]",                                             
```

Comment:
This needs to be your telegram username, it can be found under the settings -> profile -> username section of your telegram app